scripts/enable_gitlab.sh

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
```

Comment:
Can we please agree on one pattern to name these things? All but one of these scripts are named using `-` instead of `_`, we really should have a convention for this, I'd go with `-`